dataflow/src/main/java/org/checkerframework/dataflow/cfg/DOTCFGVisualizer.java

```diff
@@ -193,6 +194,28 @@ protected String dotOutputFileName(UnderlyingAST ast) {
             srcLoc.append(")::");
             srcLoc.append(((JCTree) cfgMethod.getMethod()).pos);
             srcLoc.append(">");
+        } else if (ast.getKind() == UnderlyingAST.Kind.LAMBDA) {
+            CFGLambda cfgLambda = (CFGLambda) ast;
+            String enclosingClassName = cfgLambda.getEnclosingClass().getSimpleName().toString();
+            String enclosingMethodName = cfgLambda.getEnclosingMethod().getName().toString();
+            int hashCode = cfgLambda.getCode().hashCode();
+            outFile.append(enclosingClassName);
+            outFile.append("-");
+            outFile.append(enclosingMethodName);
+            outFile.append("-");
+            outFile.append(hashCode);
+
+            srcLoc.append("<");
+            srcLoc.append(enclosingClassName);
+            srcLoc.append("::");
+            srcLoc.append(enclosingMethodName);
+            srcLoc.append("(");
+            srcLoc.append(cfgLambda.getEnclosingMethod().getParameters());
+            srcLoc.append(")::");
+            srcLoc.append(hashCode);
```

Comment:
Maybe the hashCode isn't necessary here - in ARBITRARY_CODE it's also not included.